lstchain/data/catB_camera_calibration_param.json

```diff
@@ -3,13 +3,13 @@
   "CatBCalibrationHDF5Writer": {
     "one_event": false,
     "calibration_product": "LSTCalibrationCalculator",
-    "log_level": "DEBUG"
+    "log_level": "INFO"
   },
   "LSTCalibrationCalculator": {
     "squared_excess_noise_factor": 1.222,
     "use_scaled_low_gain": true,
+    "hg_lg_ratio": 17.4,
```

Comment:
Can you say a few words about the change in this value?

Reply:
this is the correct ratio high gain/low gain (which is also in the default of the trailet). I honestly do not remember why the value in the json was 1 ...